client/ayon_maya/api/lib_rendersettings.py

```diff
@@ -197,22 +197,22 @@ def _set_redshift_settings(self):
         # update the AOV list
         mel.eval("redshiftUpdateActiveAovList")
 
-        rs_p_engine = redshift_render_presets["primary_gi_engine"]
-        rs_s_engine = redshift_render_presets["secondary_gi_engine"]
-
-        if int(rs_p_engine) or int(rs_s_engine) != 0:
-            cmds.setAttr("redshiftOptions.GIEnabled", 1)
-            if int(rs_p_engine) == 0:
-                # reset the primary GI Engine as default
-                cmds.setAttr("redshiftOptions.primaryGIEngine", 4)
-            if int(rs_s_engine) == 0:
-                # reset the secondary GI Engine as default
-                cmds.setAttr("redshiftOptions.secondaryGIEngine", 2)
-        else:
-            cmds.setAttr("redshiftOptions.GIEnabled", 0)
-
-        cmds.setAttr("redshiftOptions.primaryGIEngine", int(rs_p_engine))
-        cmds.setAttr("redshiftOptions.secondaryGIEngine", int(rs_s_engine))
+        # Irradiance Point cloud: 2 (only available for secondary GI engine)
+        # Irradiance Cache: 3 (only available for primary GI engine)
+        # Brute Force: 4
+        rs_p_engine = int(redshift_render_presets["primary_gi_engine"])
+        rs_s_engine = int(redshift_render_presets["secondary_gi_engine"])
+        if rs_p_engine == 0:
+            # reset the primary GI Engine as default: Brute Force
+            rs_p_engine = 4
```

Comment:
I am thinking if it would be better to use the current setting for primary/secondary GI Engines to allow some freedom for users to use their own preferences(just in case they want to test something with that).
```suggestion
            rs_p_engine = int(cmds.getAttr("redshiftOptions.primaryGIEngine"))
```

Reply:
>I'd have expected myself having a separate "GI enabled" checkbox in settings makes more sense - so it's explicit about what is happening.

I agree with this as it would ease the confusion, if the purpose of `None` is not for users to set it by their own.